_schemaV2.graphql

```diff
@@ -4969,6 +4972,9 @@ enum CommerceOrderSourceEnum {
 
   # The order was originated on a conversation
   inquiry
+
+  # The order was originated from a private sale
+  private_sale
```

Comment:
this one is not related but picked up. i guess we haven't updated the schema since we added this source field.